static/images/globe_logo.svg

```diff
@@ -1,7 +1,7 @@
 <svg width="85" height="85" viewBox="0 0 85 85" fill="none" xmlns="http://www.w3.org/2000/svg">
 <circle cx="42.5" cy="42.5" r="42.5" fill="#C9D4D9"/>
 <mask id="mask0" mask-type="alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="85" height="85">
-<circle cx="42.5" cy="42.5" r="42.5" fill="#2591C2"/>
+<circle cx="42.5" cy="42.5" r="42.5" fill="#205572"/>
```

Comment:
In my opinion, there is no need to change the color in the log itself.

In the issue, I thought more about applying the color schema if possible to the rest of the page.